build.gradle

```diff
@@ -6,25 +6,26 @@ plugins {
     id 'com.jfrog.artifactory'
     id 'org.javamodularity.moduleplugin'
     id 'com.github.johnrengelman.shadow'
-    id 'biz.aQute.bnd.builder' apply false
     id 'com.dorongold.task-tree'
+    id 'biz.aQute.bnd.builder' apply false
+    id "org.jetbrains.kotlin.jvm" apply false
 }
 
+sourceCompatibility = '11'
+targetCompatibility = '11'
```

Comment:
Why move these?

Reply:
Because I reasoned that these ought to apply globally rather than just subprojects.

Reply:
But _do_ these now apply globally? Or do they only apply to the root project?

Reply:
I will put them back in the subproject.